### ISSUE TITLE
Importing Server List crashes Game

### ISSUE THREAD

**Issue by GummiBoat** [Wurst-Imperium/Wurst-Client#848]
```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// Everything's going to plan. No, really, that was supposed to happen.

Time: 01.06.15 00:15
Description: Updating screen events

java.lang.NullPointerException: Updating screen events
	at net.minecraft.client.gui.GuiMultiplayer.handleMouseInput(GuiMultiplayer.java:111)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1971)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1300)
	at net.minecraft.client.Minecraft.run(Minecraft.java:411)
	at net.minecraft.client.main.Main.main(Main.java:127)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at net.minecraft.client.gui.GuiMultiplayer.handleMouseInput(GuiMultiplayer.java:111)
	at net.minecraft.client.gui.GuiScreen.handleInput(GuiScreen.java:560)

-- Affected screen --
Details:
	Screen name: net.minecraft.client.gui.GuiMultiplayer
Stacktrace:
	at net.minecraft.client.Minecraft.runTick(Minecraft.java:1971)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1300)
	at net.minecraft.client.Minecraft.run(Minecraft.java:411)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 1.14.1 (latest: 1.14.1)
	Operating System: Windows 7 (x86) version 6.1
	Java Version: 1.8.0_45, Oracle Corporation
	Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
	Memory: 231402216 bytes (220 MB) / 449794048 bytes (428 MB) up to 523501568 bytes (499 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 13, tallocated: 95
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: GeForce GT 630/PCIe/SSE2 GL version 4.4.0, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: [faithful32pack.zip, CodeCrafted Custom.zip]
	Current Language: English (UK)
	Profiler Position: N/A (disabled)
```
Happened while I imported a Server List with 61 Entries. Dunno if it's just my shitty Computer.

Tested it now more specificly, crash appears only >= 40 entries. But after restarting the game, all Servers show up.

EDIT: I just noticed, that the Crash Report Changed.. I'll play around a bit.

**Comment by GummiBoat** [comment]
Okay, so I did following: Server Finder -> Clean Up (Standard Settings) -> Exporting -> Clean Up (Removing all Servers) -> Importing.

And then this different Crash occured.

```
---------------- Wurst Client Crash Report ----------------
---- Please report this crash at: wurst-client.tk/bugs ----
// There are four lights!

Time: 01.06.15 00:30
Description: Rendering screen

java.lang.IndexOutOfBoundsException: Index: 34, Size: 0
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at net.minecraft.client.gui.ServerSelectionList.getListEntry(ServerSelectionList.java:45)
	at net.minecraft.client.gui.GuiListExtended.func_178040_a(GuiListExtended.java:36)
	at net.minecraft.client.gui.GuiSlot.drawSelectionBox(GuiSlot.java:459)
	at net.minecraft.client.gui.GuiSlot.drawScreen(GuiSlot.java:260)
	at net.minecraft.client.gui.GuiMultiplayer.drawScreen(GuiMultiplayer.java:499)
	at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1198)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1326)
	at net.minecraft.client.Minecraft.run(Minecraft.java:411)
	at net.minecraft.client.main.Main.main(Main.java:127)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Stacktrace:
	at java.util.ArrayList.rangeCheck(Unknown Source)
	at java.util.ArrayList.get(Unknown Source)
	at net.minecraft.client.gui.ServerSelectionList.getListEntry(ServerSelectionList.java:45)
	at net.minecraft.client.gui.GuiListExtended.func_178040_a(GuiListExtended.java:36)
	at net.minecraft.client.gui.GuiSlot.drawSelectionBox(GuiSlot.java:459)
	at net.minecraft.client.gui.GuiSlot.drawScreen(GuiSlot.java:260)
	at net.minecraft.client.gui.GuiMultiplayer.drawScreen(GuiMultiplayer.java:499)

-- Screen render details --
Details:
	Screen name: net.minecraft.client.gui.GuiMultiplayer
	Mouse location: Scaled: (262, 178). Absolute: (1050, 424)
	Screen size: Scaled: (400, 285). Absolute: (1600, 1138). Scale factor of 4
Stacktrace:
	at net.minecraft.client.renderer.EntityRenderer.updateCameraAndRender(EntityRenderer.java:1198)
	at net.minecraft.client.Minecraft.runGameLoop(Minecraft.java:1326)
	at net.minecraft.client.Minecraft.run(Minecraft.java:411)
	at net.minecraft.client.main.Main.main(Main.java:127)

-- System Details --
Details:
	Minecraft Version: 1.8
	Wurst Client Version: 1.14.1 (latest: 1.14.1)
	Operating System: Windows 7 (x86) version 6.1
	Java Version: 1.8.0_45, Oracle Corporation
	Java VM Version: Java HotSpot(TM) Client VM (mixed mode), Oracle Corporation
	Memory: 162721976 bytes (155 MB) / 290824192 bytes (277 MB) up to 523501568 bytes (499 MB)
	JVM Flags: 6 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xmx512M -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M
	IntCache: cache: 0, tcache: 0, allocated: 0, tallocated: 0
	Launched Version: Wurst
	LWJGL: 2.9.1
	OpenGL: GeForce GT 630/PCIe/SSE2 GL version 4.4.0, NVIDIA Corporation
	GL Caps: Using GL 1.3 multitexturing.
Using GL 1.3 texture combiners.
Using framebuffer objects because OpenGL 3.0 is supported and separate blending is supported.
Shaders are available because OpenGL 2.1 is supported.
VBOs are available because OpenGL 1.5 is supported.

	Using VBOs: No
	Is Modded: Very likely; Jar signature invalidated
	Type: Client (map_client.txt)
	Resource Packs: [faithful32pack.zip, CodeCrafted Custom.zip]
	Current Language: English (UK)
	Profiler Position: N/A (disabled)
```

**Comment by Alexander01998** [comment]
Lists in Minecraft do weird things sometimes. Should be fixed now.
